server/src/main/java/org/apache/uniffle/server/ShuffleServerMetrics.java

```diff
@@ -131,6 +138,19 @@ public class ShuffleServerMetrics {
       "topN_of_on_localfile_data_size_for_app";
   public static final String TOPN_OF_ON_HADOOP_DATA_SIZE_FOR_APP =
       "topN_of_on_hadoop_data_size_for_app";
+  public static final double[] BLOCK_SIZE_BUCKETS =
+      new double[] {
```

Comment:
Could you make this configurable like event_size_level?

Reply:
You could provide the number of buckets (10), the min (32 * KiB), and a factor (2).